external/provider.sh

```diff
@@ -107,7 +107,7 @@ function getFullOpenJ9Image() {
   local JDK_VERSION="${1:-0}"
   local jreSdk="${2:-jdk}"
   if [ -z "${EXTERNAL_AQA_IMAGE}" ]; then
-    echo "docker.io/library/ibm-semeru-runtimes:open-${JDK_VERSION}-$jreSdk"
+    echo "icr.io/appcafe/ibm-semeru-runtimes:open-${JDK_VERSION}-$jreSdk-jammy"
```

Comment:
Is it possible to use a variable for this echo? If the actual image changes in the future, this echo message will be misleading.